packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -180,8 +180,16 @@ function BlockPopoverInbetween( {
 		const observer = new window.MutationObserver( forcePopoverRecompute );
 		observer.observe( previousElement, { attributes: true } );
 
+		const zoomedOutObserver = new window.MutationObserver(
+			forcePopoverRecompute
+		);
+		zoomedOutObserver.observe( previousElement.closest( 'html' ), {
```

Comment:
I think this might be a bit too aggressive to recompute the popover position on every attribute change. In addition, I don't think it's immediately clear why we want to observe only the `html` element for the zoom out mode. Another idea would be using `ResizeObserver` in a similar way, which I believe we already have one somewhere?

Reply:
Noting that we are observing the `html` element _only_. This is because [the default for the `subtree` option of the `.observe()` method is `false`](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/observe#subtree).

> Set to true to extend monitoring to the entire subtree of nodes rooted at target. All of the other properties are then extended to all of the nodes in the subtree instead of applying solely to the target node.

Therefore this should only be recomputing if attributes change on the `html` node itself. How often does that realistically happen? Probably quite infrequently. 

I could be wrong but it's worth noting.

Reply:
Yes, I mentioned this above:

> I don't think it's immediately clear why we want to observe **only** the html element for the zoom out mode.

A comment above would be nice so people won't accidentally break it.

Reply:
Ah I see. I misinterpreted your comment. Comment always good